src/ActiveLogin.Identity.Swedish/SwedishPersonalIdentityNumber.fs

```diff
@@ -88,8 +124,15 @@ module Hints =
             | false -> months / 12 |> Some
         | false -> None
 
+    /// Get the age of the person according to the date in the personal identity number.
+    /// Not always the actual date of birth due to the limited quantity of personal identity numbers per day.
+    /// <param name="pin"> A SwedishPersonalIdentityNumber</param>
     let getAgeHint pin = getAgeHintOnDate DateTime.UtcNow pin
 
+    /// Gender (juridiskt kön) in Sweden according to the last digit of the birth number in the personal identity number.
+    /// Odd number: Male
+    /// Even number: Female
+    /// <param name="pin"> A SwedishPersonalIdentityNumber</param>

```

Comment:
The `<param name="pin"> A...` contains a prefixed whitespace, probably the same in C#, can you fix that?

Reply:
Couldn't find it in the C# facade, but we did have the same issue in several places in the F#-file.